src/features/MoviesBrowser/PopularMovies/useMoviesData.js

```diff
@@ -1,16 +1,17 @@
 import { useState, useEffect } from "react";
 
-const discoverMovie =
-  "https://api.themoviedb.org/3/discover/movie?include_adult=false&include_video=false&language=en-US&page=1&sort_by=popularity.desc";
-
-const usedFetchAddress = discoverMovie;
-
-export const useMoviesData = () => {
+export const useMoviesData = (pageWanted) => {
   const [moviesData, setMoviesData] = useState({
     status: "loading",
+    page: pageWanted,
     results: {},
```

Comment:
shouldn't there be a board here?

Reply:
You meant Array I suppose, it's not that important 👌